.github/pull_request_template.md

```diff
@@ -9,11 +9,21 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+These should be followed unless you're dealing with an emergency. In an emergency, use your best judgment. 
```

Comment:
Consider adding: 
`If some steps are not applicable, remove and explain why rather than just leave those checkmarks blank`